File: CPKINGDOM/CPKINGDOM.Portal/ClientApp/src/app/models/item.ts
export class Item {
    id: number;
    barcode: string;
    name: string;
    description: string;
    srp: number;
    categoryId: number;
    categoryName: string;
    brandId: number;
    brandName: string;
    reorderPoint: number;
    criticalLevel: number;
    constructor() {
        this.id = 0;
        this.barcode = '';
        this.name = '';
        this.description = '';
        this.srp = 0;
        this.categoryId = 0;
        this.categoryName = '';
        this.brandId = 0;
        this.brandName = '';
        this.reorderPoint = 0;
        this.criticalLevel = 0;
    }
}

File: CPKINGDOM/CPKINGDOM.Api/Models/Item.cs
namespace CPKINGDOM.Api.Models
{
    public class Item
    {
        public int Id { get; set; }
        public string Barcode { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Srp { get; set; }
        public int CategoryId { get; set; }
        public string CategoryName { get; set; }
        public int BrandId { get; set; }
        public string BrandName { get; set; }
        public int ReorderPoint { get; set; }
        public int CriticalLevel { get; set; }
    }
}
